runtime/zeitgeist/src/parameters.rs

```diff
@@ -290,6 +292,8 @@ parameter_types! {
     /// Minimum amount of the multiplier. The test `multiplier_can_grow_from_zero` ensures
     /// that this value is not too low.
     pub MinimumMultiplier: Multiplier = Multiplier::saturating_from_rational(1, 1_000_000u128);
+    /// Maximum amount of the multiplier.
+    pub MaximumMultiplier: Multiplier = Bounded::max_value();
```

Comment:
Can you send me a reference, where you found this configuration?

Reply:
Based on https://github.com/paritytech/substrate/blob/master/frame/transaction-payment/src/lib.rs#L136